src/components/ManifestDetailSidePanel/ManifestDetailSidePanel.tsx

```diff
@@ -0,0 +1,199 @@
+        <h3 tabIndex={isSuccess ? 0 : -1} ref={drawerRef}>
+          {name}
+        </h3>
+        <h4>Details</h4>
+        <Grid>
+          <GridItem span={6}>
+            <p>
+              <strong>Name</strong>
+            </p>
+          </GridItem>
+          <GridItem span={6}>
+            <p>{name}</p>
+          </GridItem>
+
+          <GridItem span={6}>
+            <p>
+              <strong>Type</strong>
+            </p>
+          </GridItem>
+          <GridItem span={6}>
+            <p>Satellite {version}</p>
+          </GridItem>
+
+          <GridItem span={6}>
+            <p>
+              <strong>UUID</strong>
+            </p>
+          </GridItem>
+          <GridItem span={6}>
+            <p>{uuid}</p>
+          </GridItem>
+        </Grid>
+
+        <h4>Subscriptions</h4>
+
+        <Grid>
+          <GridItem span={6}>
+            <p>
+              <strong>
+                Simple content access
+                <SCAInfoIconWithPopover />
+              </strong>
+            </p>
+          </GridItem>
+          <GridItem span={6}>
+            <p>{contentAccessMode}</p>
+          </GridItem>
+          <GridItem span={6}>
+            <p>
+              <strong>Subscriptions</strong>
+            </p>
+          </GridItem>
+          <GridItem span={6}>
+            <Button
+              variant="link"
+              onClick={openCurrentEntitlementsListFromPanel}
+              className="manifest-details-open-entitlements-button"
+            >
+              {entitlementsAttachedQuantity}
+            </Button>
+          </GridItem>
+        </Grid>
+
+        <h4>History</h4>
+        <Grid>
+          <GridItem span={6}>
+            <p>
+              <strong>Created</strong>
+            </p>
+          </GridItem>
+          <GridItem span={6}>
+            <p>{formatDate(createdDate)}</p>
+          </GridItem>
+
+          <GridItem span={6}>
+            <p>
+              <strong>Created by</strong>
+            </p>
+          </GridItem>
+          <GridItem span={6}>
+            <p>{createdBy}</p>
+          </GridItem>
+
+          <GridItem span={6}>
+            <p>
+              <strong>Last modified date</strong>
+            </p>
+          </GridItem>
+          <GridItem span={6}>
+            <p>{formatDate(lastModified)}</p>
+          </GridItem>
+        </Grid>
+        <Button variant="tertiary">Export manifest</Button>
+        <p className="manifest-details-delete-text">
+          Deleting a subscription allocation is <strong>STRONGLY</strong> discouraged. This action
+          should only be taken in extreme circumstances or for debugging purposes
+        </p>
+        <Button variant="tertiary">Delete manifest</Button>
+      </div>
+    );
+  };
+
+  const Loading = () => (
+    <div
+      className="manifest-detail-drawer-loading"
+      aria-label="Loading Manifest Details"
+      tabIndex={isSuccess ? 0 : -1}
```

Comment:
Updated with that; good idea! looks much better now.  

https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView with refs works well. I had to add a ref to the page title, because once I fixed the animation of the side drawer, trying to tie the page scroll to the (moving) loading spinner resulted in really glitchy behavior.

Reply:
Odd.